roles/deploy/tasks/cleanup.yml

```diff
@@ -0,0 +1,6 @@
+---
+# Remove all dirs, but keep the current version and from the rest the most recent one.
+- name: Clean up old releases
+  shell: ls -td $(echo "{{ component_dir_name }}" | cut -d "-" -f -2)-* | grep -v $(readlink /opt/www/{{ component_vhost_name }}) | tail -n +3 | xargs --no-run-if-empty rm -rv
```

Comment:
This is a complex oneliner, not very readable, adds too complexity
If something unexpected happens, it will be hard to debug.
It does not log what it does.

Use external script? split in multiple actions? more comments?

Reply:
What is the the effect on dev VM

Reply:
Added comments to explain the oneliner
Excluded the task if on a dev VM